prime-angular-frontend/src/app/modules/site-registration/pages/administrator-page/administrator-page.component.ts

```diff
@@ -45,6 +48,7 @@ export class AdministratorPageComponent extends AbstractEnrolmentPage implements
 
     this.title = route.snapshot.data.title;
     this.routeUtils = new RouteUtils(route, router, SiteRoutes.MODULE_PATH);
+    this.formSubmittingEvent = new Subject<void>();
```

Comment:
Is there a reason to have this as a `Subject`?  With a form submission you can't unring the bell once you submit.  Feels like this could be `formSubmitted: boolean` since `Input` bindings are reactive which would be defaulted to false and once changed can be picked up in the `ngOnChange` lifecycle hook if you need to do perform tasks on the change.

Reply:
`ngOnChange` gets invoked, but it has a `SimpleChanges` param that shows whether a value has changed.  You don't have to worry about it being a performance issue since it gets invoked whether you use it or not with it being a lifecycle hook.

Reply:
By using an observable you're saying that subsequent events might change the value, but once you've submitted once it should remain true forever so the observable isn't being used as you would expect where data changes overtime.

Reply:
It's a Subject holding void value and the subscriber contact-form component only cares about the form submitting event triggering. This is an alternative of onSubmit hook for child component, not meant to pass true/false value downstream. Will try out the ngOnChange hook approach.